coordinatorlib/src/main/scala/com/socrata/datacoordinator/secondary/sql/SqlSecondaryManifest.scala

```diff
@@ -383,4 +385,30 @@ class SqlSecondaryManifest(conn: Connection) extends SecondaryManifest {
       }
     }
   }
+
+  def lockResync(datasetId: DatasetId, storeId: String, groupName: String): Int = {
+    using(conn.prepareStatement("INSERT INTO resync(dataset_system_id, store_id, group_name) values(?, ?, ?)")) { stmt =>
+      val savepoint = conn.setSavepoint()
```

Comment:
It will be ok-ish.  Its group default to "" and will be mixed with others which do not set the group.  We probably should add group name unless you have a reason not to.